docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,451 @@
+      - ["name",                      "event_hub_connection",      "storage_connection",                                      "initial_position", "decorate_events"]
+      - ["insights-operational-logs", "Endpoint=sb://example1...", "DefaultEndpointsProtocol=https;AccountName=example1....", "HEAD",             "true"]
+      - ["insights-metrics-pt1m",     "Endpoint=sb://example2...", "DefaultEndpointsProtocol=https;AccountName=example2....", "TAIL",             "true"]
+      - ["insights-logs-errors",      "Endpoint=sb://example3...", "DefaultEndpointsProtocol=https;AccountName=example3....", "TAIL",             "false"]
+      - ["insights-operational-logs", "Endpoint=sb://example4...", "DefaultEndpointsProtocol=https;AccountName=example4....", "HEAD",             "true"]
+-----
+
+===== Map input options to the logstash.yml file
+
+The configurations in the `logstash.yml` file must be in YAML format. 
+
+Example:
+
+["source","shell",subs="attributes"]
+-----
+var.input.azure_event_hubs.event_hubs:
+  - ["name",                      "event_hub_connection",      "storage_connection",                                      "initial_position", "decorate_events"]
+  - ["insights-operational-logs", "Endpoint=sb://example1...", "DefaultEndpointsProtocol=https;AccountName=example1....", "HEAD",             "true"]
+  - ["insights-metrics-pt1m",     "Endpoint=sb://example2...", "DefaultEndpointsProtocol=https;AccountName=example2....", "TAIL",             "true"]
+-----
+
+[[azure_config_options]]
+===== Configuration options
+
+NOTE: All Event Hubs options are common to both basic and advanced
+configurations, with the following exceptions. The basic configuration uses
+`event-hub-connections`. The the advanced configuration uses `event_hubs` and
+`event_hub_connection`.
+
+[id="plugins-{type}s-{plugin}-event_hubs"]
+===== `event_hubs`
+* Value type is <<array,array>>
+* No default value
+* Ignored for basic configuration
+* Required for advanced configuration 
+
+Defines the Event Hubs to be read. An array of hashes where each entry is a
+hash of the Event Hub name and its configuration options.   
+
+
+[id="plugins-{type}s-{plugin}-event_hub_connections"]
+===== `event_hub_connections`
+* Value type is <<array,array>>
+* No default value
+* Required for basic configuration
+
+List of connection strings that identifies the Event Hubs to be read. Connection
+strings include the EntityPath for the Event Hub. 
+
+The `event_hub_connections` option is defined
+per Event Hub. All other configuration options are shared among Event Hubs.
+
+
+[id="plugins-{type}s-{plugin}-event_hub_connection"]
+===== `event_hub_connection`
+* Value type is <<string,string>>
+* No default value
+* Valid only for advanced configuration
+
+Connection string that identifies the Event Hub to be read. Advanced
+configuration options can be set per Event Hub. This option modifies
+`event_hub_name`, and should be nested under it. (See sample.) This option
```

Comment:
I'm just skimming for issues, but it seems like "See sample"  is an incomplete sentence and perhaps needs a link